bitrise.yml

```diff
@@ -86,14 +85,6 @@ workflows:
             docker-compose rm -f -v
             docker-compose build
 
-  client-build-website:
-    envs:
-      - NODE_ENV: "prod"
-      - MODE: "WEBSITE"
-      - ANALYTICS: true
-    after_run:
-    - setup-client
-
```

Comment:
this moved to: https://app.bitrise.io/app/1686da85b5935fd6/workflow_editor#!/workflows?workflow_id=website-deploy